tests/unit/backend/data/test_LocalDataService.py

```diff
@@ -1827,19 +1830,19 @@ def test_readDetectorState():
     localDataService = LocalDataService()
     localDataService._readPVFile = mock.Mock()
 
-    pvFileMock = mock.Mock()
-    # 1X: seven required `readDetectorState` log entries:
-    #   * generated `DetectorInfo` matches that from 'inputs/calibration/CalibrationParameters.json'
-    pvFileMock.get.side_effect = [
-        [1],
-        [2],
-        [1.1],
-        [1.2],
-        [1],
-        [1.0],
-        [2.0],
-    ]
-    localDataService._readPVFile.return_value = pvFileMock
+    pvFile = {
```

Comment:
You can, and probably should _centralize_ this pvFile mock as:
`def mockPVFile(detectorState: DetectorState) -> Dict[str, Any]:
...`